### PR TITLE
chore: codepush remove native check

### PR DESCRIPTION
**Pull request by brainbicycle**
This PR resolves [] 

### Description

This check is more trouble than it is worth, instead:
- When merging to main we will check against a hash that is stored in ci, if it is different notify that native code has changed and store the new hash, if the same deploy to codepush

For hot fixes we will need to make sure codepush updates are compatible otherwise go through the normal update process, this is also solvable without requiring this hash to be checked in and updated by devs. 

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Remove native code ci checks and related scripts - Brian 

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 34c35ec0eb6fa56d2e652e5b8fb544f404092b7a
</p>

File: .gitignore [hunk 1]
```diff
@@ -87,8 +87,8 @@ tsc_raw.log
 data/complete.queryMap.json
 
 # native code hash
-native-code-checksum.hash
```
Comment (brainbicycle) [inline .gitignore:90]: I think this is quite old and I don't see it generate anywhere 🤷

**Review by MrSltun — APPROVED** [review]
✨✨

File: .gitignore (hunk 1 above)
Comment (araujobarret) [reply .gitignore:90]: I guess we have some legacy stuff from time to time we will get rid of 🧹

**Comment by araujobarret** [comment]
Q about jq, locally when I run the setup script it tries to install, I guess some package already requires it and we don't need it anymore, right?

**Comment by brainbicycle** [comment]
> Q about jq, locally when I run the setup script it tries to install, I guess some package already requires it and we don't need it anymore, right?

a couple of our local scripts still use it, the deploy betas scripts it looks like and the init-metaflags

**Pull request merged**